app/routes/flush.py

```diff
@@ -63,6 +63,9 @@ def flush_data() -> Response:
                 contextvars.bind_contextvars(schema_name=schema_name)
             if schema_url := metadata.schema_url:
                 contextvars.bind_contextvars(schema_url=schema_url)
+            if cir_instrument_id := metadata.cir_instrument_id:
+                contextvars.bind_contextvars(cir_instrument_id=cir_instrument_id)
```

Comment:
Make the helper public and re-use `_bind_contextvars_schema_from_metadata`?. Worth moving to different helper file.

Reply:
Put in a shared utilities method 👍